### PR TITLE
Update messages.po

### PR DESCRIPTION
**Pull request by MattiaDes**
Updated messages.po for the Italian language. There are still translations to be made, I'll do them shortly.

### CODE REVIEW

**Comment by MattiaDes** [comment]
I signed it!

**Comment by MattiaDes** [comment]
I signed it!

**Comment by MattiaDes** [comment]
I signed it!

File: translations/it/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1020,7 +1045,7 @@ msgstr ""
 
 #: /content/includes/home.yaml
 msgid "What AMP Provides"
-msgstr ""
+msgstr "Cosa Fornisce APM"
```
Comment (bpaduch) [inline translations/it/LC_MESSAGES/messages.po]: Correct APM to AMP

**Comment by MattiaDes** [comment]
Yes, thanks 👍 done

**Comment by bpaduch** [comment]
Hi MattiaDes,

The CLA can be tricky.  Can you make sure the email address you used for this pull request is associated with your GitHub account - https://github.com/settings/emails.

Thank you.

**Comment by MattiaDes** [comment]
Hi,
Yes, I updated the email that I use on my GitHub account. 
Do I have to do more?

**Comment by bpaduch** [comment]
Your first commit had a different email address than what's on your GitHub account, it starts with `habb***@live.it`.  Can you add that to your GitHub account?

**Comment by MattiaDes** [comment]
Ah ok, thank you for the information. I added that email into my GitHub account. (But I didn't put it as primary)

Let me know if it's all ok now. Thank you.

**Pull request merged**